packages/builder/cypress/integration/addMultiOptionDatatype.spec.js

```diff
@@ -0,0 +1,40 @@
+context("Add Multi-Option Datatype", () => {
+    before(() => {
+      cy.login()
+      cy.createTestApp()
+    })
+
+    it("should create a new table, with data", () => {
+        cy.createTable("Multi Data")
+        cy.addColumn("Multi Data", "Test Data", "Multi-select", "1\n2\n3\n4\n5")
+        cy.addRowMultiValue(["1", "2", "3", "4", "5"])
+    })
+
+    it ("should add form with multi select picker, containing 5 options", () => {
+        cy.navigateToFrontend()
+        cy.wait(500)
+        // Add data provider
+        cy.get(`[data-cy="category-Data Provider"]`).click()
```

Comment:
nice that you have used the `data-cy` selectors here - it makes the tests much easier to read 👍